pandas/tests/io/test_parquet.py

```diff
@@ -410,11 +410,24 @@ def test_write_multiindex(self, pa):
         check_round_trip(df, engine)
 
     def test_write_column_multiindex(self, engine):
-        # column multi-index
+        # Not able to write column multi-indexes with non-string column names.
         mi_columns = pd.MultiIndex.from_tuples([("a", 1), ("a", 2), ("b", 1)])
         df = pd.DataFrame(np.random.randn(4, 3), columns=mi_columns)
         self.check_error_on_write(df, engine, ValueError)
 
+    def test_write_column_multiindex_string(self, pa):
```

Comment:
Can you include the issue number or URL as a comment?

Reply:
Added issue number as a comment on test in latest commit.